Resources/Prototypes/Datasets/ion_storm.yml

```diff
@@ -12,7 +12,6 @@
   - BORED
   - BOUNCING
   - BRASS
-  - BROWN
```

Comment:
why?

Reply:
In the issue, @wrexbe said there are can be racist outcomes, so im mitigating that

Reply:
You might want to get rid of "black" a few lines up too, in that case.